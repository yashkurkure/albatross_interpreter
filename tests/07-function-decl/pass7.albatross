var f int := 0;

fun f int () {
  return 1;
}

return 2;
