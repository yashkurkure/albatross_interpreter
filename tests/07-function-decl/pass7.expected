Variable declared "f" type int
Function declared "f" returns int
